tt_metal/hw/ckernels/wormhole_b0/common/inc/cmath_common.h

```diff
@@ -145,9 +140,7 @@ inline uint32_t get_dest_buffer_base()
 inline void wait_math_semaphores()
 {
     // wait while math semaphore is on max, no room to write math results
-    DEBUG_STATUS('W', 'M', 'S', 'W');
     TTI_SEMWAIT(p_stall::STALL_MATH|p_stall::STALL_SFPU, semaphore::t6_sem(semaphore::MATH_PACK), p_stall::STALL_ON_MAX);
-    DEBUG_STATUS('W', 'M', 'S', 'D');
```

Comment:
why were the DEBUG_STATUS markers removed?

Reply:
@pgkeller I synced with Reem and she noted that eventually llks will be shared between buda and metal. Since buda doesn't use watcher, the common part will not have debug status. She will instead have metal wrappers with the debug status macros.